bom/runtime/pom.xml

```diff
@@ -878,11 +878,11 @@
       <version>${xtext.version}</version>
       <scope>compile</scope>
     </dependency>
-    <dependency>
+    <!--dependency>
       <groupId>log4j</groupId>
       <artifactId>log4j</artifactId>
       <scope>compile</scope>
-    </dependency>
+      </dependency -->
```

Comment:
I would remove this if it is unneccesary and not required for running itests and debugging in Eclipse.